frontend/src/layout/navigation-3000/sidepanel/sidePanelStateLogic.tsx

```diff
@@ -55,10 +56,12 @@ export const sidePanelStateLogic = kea<sidePanelStateLogicType>([
     listeners(({ actions, values }) => ({
         // NOTE: We explicitly reference the actions instead of connecting so that people don't accidentally
         // use this logic instead of sidePanelStateLogic
-        openSidePanel: () => {
+        openSidePanel: ({ tab }) => {
+            posthog.capture('activation sidebar opened', { tab })
```

Comment:
I think this will fire every time a side panel is opened, no? Seems like this isn't only for the activation panel. We prob need to filter on the tab.

Reply:
Ah yes, I will update event to `sidebar opened` opened, nice catch.